go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/openshift/ocm-agent
 
-go 1.19
+go 1.21
```

Comment:
Do we have to bump Go to bump the OCM sdk too?

Reply:
no, I just upgraded my local go installation to 1.21 and I think it might be the Go toolchain automatically updated the `go` to reflect the new version. Good catch, I will edit it to revert to 1.19 then.